scripts/init_test_environment.py

```diff
@@ -3,7 +3,7 @@
 import sys
 
 
-PASSWORD = '1111'
```

Comment:
once I made the changes it works... The default reddit install complained about the password being too short initially.  I don't think that the adjust_subscriptions code is working **correctly**... but it doesn't throw errors anymore.

It was kind of nice to verify that most the functionality was working on my local reddit... It did create users, subreddits, posts, and chagne subscriptions for hte users.